requirements.txt

```diff
@@ -1,5 +1,5 @@
 plumbum==1.7.2
-requests>=2.25.1
+requests>=2.31.0
```

Comment:
Does it hurt to pin it to exact version of `requests`? I see this is the only dep that is not fixed